composer.json

```diff
@@ -11,7 +11,8 @@
   ],
   "require": {
     "wpackagist-plugin/wp-graphql": "^1.0",
-    "pantheon-systems/pantheon-decoupled-auth-example": "^1.0"
+    "pantheon-systems/pantheon-decoupled-auth-example": "^1.0",
+    "pantheon-systems/wp-decoupled-preview": "^1.0"
```

Comment:
@abhisekmazumdar did you test this? I would expect this to fail because we haven't published this plugin on packagist yet.

Could we work around that in the short term by adding the necessary git repo to composer.json, or do we need to publish this on packagist first?

Reply:
Yes that's what I was very confused about. I though publishing this to packagist will be done as part of this ticket. So I made the required changes.

Reply:
I think this still won't work - the package was published as pantheon-systems/decoupled-preview